modules/serverless-creating-subscriptions-kn.adoc

```diff
@@ -17,16 +17,19 @@ To create subscriptions using the `kn` CLI, ensure that:
 
 .Procedure
 
-* Create a subscription to connect a sink to a channel.
+* Create a subscription to connect a sink to a channel:
 +
 [source,terminal]
 ----
 $ kn subscription create <subscription_name> \
-  --channel <Group:Version:Kind>:<channel_name> \
-  --sink <sink_prefix>:<sink_name> \
-  --sink-reply <sink_prefix>:<sink_name> \
-  --sink-dead-letter <sink_prefix>:<sink_name>
-----
+  --channel <Group:Version:Kind>:<channel_name> \ <1>
```

Comment:
I don't think we should add the `[ ]` since everything included in a command is pretty much assumed to be literal. Happy to change the example to `  --channel <group>:<version>:<kind> <channel_name> \ <1>` though if that's acceptable?

Reply:
sounds good to me, but plese note it's `<group>:<version>:<kind>:<channel_name>` (mind the last colon before _channel_name_ ..

Reply:
Also re-added the dead letter sink as optional